cmake/abb_librwsConfig.cmake.in

```diff
@@ -4,7 +4,7 @@
 
 include(CMakeFindDependencyMacro)
 
-list(INSERT CMAKE_MODULE_PATH 0 "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+list(INSERT CMAKE_MODULE_PATH 0 "${@PROJECT_NAME@_DIR}/../../../share/@PROJECT_NAME@/cmake")
```

Comment:
This proposed solution is quite error prone, as they assume that `abb_librws_DIR` is defined when the `abb_librwsConfig.cmake` is evaluated and they hardcoded the relative position between this file and the prefix, that would break if somebody changes the installation location of this file (as done for example by package managers such as vcpkg). 

However, it is not clear why this is change is necessary. Perhaps the `${CMAKE_CURRENT_SOURCE_DIR}` is not the correct variable, as it is not evalute to the location of `abb_librwsConfig.cmake` when this file is parsed? In that case, probably the best solution is to simply use [`CMAKE_CURRENT_LIST_DIR`](https://cmake.org/cmake/help/v3.5/variable/CMAKE_CURRENT_LIST_DIR.html) instead.

Reply:
The same approach was used in cmake_modules as seen [here](https://github.com/ros/cmake_modules/blob/0.5-devel/cmake/cmake_modules-extras.cmake.installspace.in). 
However, I just validated that CMAKE_CURRENT_LIST_DIR works also, so I can replace my PR with that if @gavanderhoorn agrees as well.

Reply:
Yes, let's change it to `CMAKE_CURRENT_LIST_DIR`.

`cmake_modules` is written the way it is because it's meant to be used with packages in Catkin workspaces. Or at least, that's what I've always understood.

The relative paths work there, but we can't assume it's always true.